tests/components/squeezebox/test_binary_sensor.py

```diff
@@ -1,34 +1,35 @@
 """Test squeezebox binary sensors."""
 
-import copy
+from copy import deepcopy
 from unittest.mock import patch
 
 from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers import entity_registry as er
 
-from . import FAKE_QUERY_RESPONSE, setup_mocked_integration
+from .conftest import FAKE_QUERY_RESPONSE
+
+from tests.common import MockConfigEntry
 
 
 async def test_binary_sensor(
     hass: HomeAssistant,
-    entity_registry: er.EntityRegistry,
+    config_entry: MockConfigEntry,
 ) -> None:
     """Test binary sensor states and attributes."""
-
-    # Setup component
     with (
         patch(
             "homeassistant.components.squeezebox.PLATFORMS",
             [Platform.BINARY_SENSOR],
         ),
         patch(
             "homeassistant.components.squeezebox.Server.async_query",
-            return_value=copy.deepcopy(FAKE_QUERY_RESPONSE),
+            return_value=deepcopy(FAKE_QUERY_RESPONSE),
```

Comment:
I don't think we need to copy?

Reply:
https://github.com/home-assistant/core/pull/125764#discussion_r1756432649

Reply:
This is because internal status update in pysqueezebox.Server updates the query value in place; it expects it to be "fresh" each time from a poll of the server, but here we are mocking the part that polls the server and returning a fixed query response.